AccountService.js

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+const headers = {Accept: 'application/ld+json, application/json'};
+export class AccountService {
+  constructor({
+    urls = {
+      base: '/accounts'
+    }
+  } = {}) {
+    this.config = {urls};
+  }
+
+  async exists({baseUrl = this.config.urls.base, email}) {
+    try {
+      await axios.get(baseUrl, {
+        params: {
+          exists: true,
+          email
+        }
+      });
+      return true;
+    } catch(e) {}
```

Comment:
if this is the correct behavior, a comment with an explanation would be helpful, or a FIXME/TODO if it's temporary.

Reply:
I did not write this but this if you l[ook in bedrock-account-http](https://github.com/digitalbazaar/bedrock-account-http/blob/551e61d012903abac47172b539c0098ced0f9eed/lib/index.js#L95)
you can see this route throws an error if the user does not exist.
Again, did not design this, so this does lead to an issue.
if for some reason the get request fails it could return a false negative.
why don't you create an issue for this and we can change it so it returns a json response with true or false? _GOOD CATCH._

Reply:
@dlongley @davidlehn do we want to address this issue in the current merge? @mattcollier  is actually correct this could lead to issues.

Reply:
ok fixed this and tested it does work however there is a new client side error in veres-wallet, if the exists function throw the loader does not stop spinning.